crates/uv-fs/src/path.rs

```diff
@@ -84,8 +84,16 @@ pub fn normalize_url_path(path: &str) -> Cow<'_, str> {
 /// Normalize a path, removing things like `.` and `..`.
 ///
 /// Source: <https://github.com/rust-lang/cargo/blob/b48c41aedbd69ee3990d62a0e2006edbb506a480/crates/cargo-util/src/paths.rs#L76C1-L109C2>
-pub fn normalize_path(path: impl AsRef<Path>) -> PathBuf {
-    let mut components = path.as_ref().components().peekable();
+///
+/// CAUTION: Assumes that the path is already absolute.
```

Comment:
Same here. Can we panic if it isn't absolute?

Reply:
Yeah.